packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -29,26 +27,22 @@ import {
     WS,
 } from '@deriv/shared';
 import { localize, Localize } from '@deriv/translations';
-import SuccessDialog from '../Components/success-dialog';
-import 'Sass/cfd.scss';
-import { connect } from '../Stores/connect';
+import SuccessDialog from '../Components/success-dialog.jsx';
+import '../sass/cfd.scss';
```

Comment:
What is the reason of removing the alias here ?

Reply:
This type alias was set on webpack config of the `cfd` package, since we are using this component in the `appstore` package, there it was unable to find this type alias and throwing import errors

cc: @farzin-deriv

Reply:
Yup, Eventually we need to remove all import aliases everywhere in every package 😬

Reply:
@hirad-deriv